pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[tool.poetry]
+name = "oipie-api-core"
+version = "0.0.1"
+description = ""
+authors = ["Your Name <you@example.com>"]
```

Comment:
Can we put some of our project examples here?
```toml
description="API to list recipes"
authors=["Jorge Bodega <jorge.bodega@acidtango.com", "Jaume Moreno <jaume@acidtango.com"]
```

Reply:
We can. Let me check.